sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 # Release History
 
-## 1.10.0b2 (Unreleased)
-
-### Features Added
+## 1.10.0 (Unreleased)
 
 ### Breaking Changes
 
-### Bugs Fixed
+> These changes do not impact the API of stable versions such as 1.9.0.
+> Only code written against a beta version such as 1.10.0b1 may be affected.
+- `validate_authority` support is not available in 1.10.0.
 
 ### Other Changes
```

Comment:
Do we need to update the changelog entry here, to just indicate that the `msal-extensions` version is no longer pinned?

Reply:
I don't think so. We did not pin msal-ex version. We allowed 0.3.x, right? :)

Reply:
Ah, true 🙂 I guess I'm just not sure if "Bumped msal-extensions version to 1.0.0" makes sense anymore -- maybe there's no longer a need to have a changelog entry for the package version